pkg/volume/local_hostpath.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2020-2021 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volume
+
+import (
+	"github.com/openebs/openebsctl/pkg/client"
+	"github.com/openebs/openebsctl/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// LocalHostpathVolInfoTemplate to store the local-hostpath volume and pvc describe related details
+	LocalHostpathVolInfoTemplate = `
+{{.Name}} Details :
+-----------------
+NAME            : {{.Name}}
+ACCESS MODE     : {{.AccessMode}}
+CAS TYPE        : {{.CasType}}
+STORAGE CLASS   : {{.StorageClass}}
+VOLUME PHASE    : {{.VolumePhase }}
+SIZE            : {{.Size}}
+CAPACITY        : {{.Capacity}}
+PV CLAIM        : {{.PVC}}
+RECLAIM POLICY  : {{.ReclaimPolicy}}
+`
+)
+
+// GetLocalHostpath returns a list of local-hostpath columes
+func GetLocalHostpath(c *client.K8sClient, pvList *corev1.PersistentVolumeList, openebsNS string) ([]metav1.TableRow, error) {
+	var rows []metav1.TableRow
+	for _, pv := range pvList.Items {
+		// Ignore all the other volumes that is not of cas-type local-hostpath
+		if pv.Labels["openebs.io/cas-type"] != util.LocalHostpath {
+			continue
+		}
+
+		name := pv.Name
+		capacity := pv.Spec.Capacity.Storage()
+		sc := pv.Spec.StorageClassName
+		attached := pv.Status.Phase
+		attachedNode, customStatus, ns, storageVersion := pv.Labels["nodeID"], "N/A", "N/A", "N/A"
```

Comment:
```suggestion
              customNs := // Get deployments from all namespace which have Label="openebs.io/component-name=openebs-localpv-provisioner"
              customStatus = ""
              version = provisionerDeployment.Label["openebs.io/version"]
		attachedNode, customStatus, ns, storageVersion := pv.Labels["nodeID"], "", "N/A", "N/A"
```

@kmova What's the best way to express the status of LocalPV volumes(plain localpv ones), should it be `"N/A"` as it's technically not-applicable or is a simple `""` slightly better on the eyes?

Reply:
The storage version for a LocalPV hostpath and a LocalPV device can be considered to be the storage version of the provisioner, we've gone forward with this assumption for ZFS-LocalPV & LVM-LocalPV too.